### ISSUE TITLE
Manually configure CLI

### ISSUE THREAD

**Issue by moetun88** [aws-amplify/amplify-cli#145]
** Which Category is your question related to? **
Auth
** What AWS Services are you utilizing? **
Cognito
** Provide additional details e.g. code snippets **
We're currently using multiple cognito user pools with a multi-step login. During the first step we do a lookup to get the right user pool and then perform authentication to that user. Is there a way to NOT bake in the user pool or AWS access keys etc and manually / programmatically enter the relevant information. 

Basically a full manual configuration at run-time. I couldn't find anything in the documentation to point me in the right direction.

**Comment by kaustavghosh06** [comment]
@moetun88 What's the flow which you went through when adding auth to your app after running the 'amplify add auth' command? Did you choose the `No, I will set up my own configuration.` option?

**Comment by kaustavghosh06** [comment]
Closing due to inactivity. Feel free to re-open if the problem still persists.
